src/vs/workbench/contrib/terminal/electron-sandbox/terminal.contribution.ts

```diff
@@ -21,5 +21,5 @@ registerSingleton(ITerminalProfileResolverService, ElectronTerminalProfileResolv
 
 // Register workbench contributions
 const workbenchRegistry = Registry.as<IWorkbenchContributionsRegistry>(WorkbenchExtensions.Workbench);
-workbenchRegistry.registerWorkbenchContribution(LocalTerminalBackendContribution, LifecyclePhase.Restored);
+workbenchRegistry.registerWorkbenchContribution(LocalTerminalBackendContribution, LifecyclePhase.Starting);
```

Comment:
This is the minimal change, but since I don't really know what these do, I don't have confidence that some small change won't cause another terminal workbench contrib to be needed at this time too.